src/pages/Book/index.js

```diff
@@ -29,7 +30,32 @@ class Book extends React.Component {
     this.getBook(isbn);
   }
 
+  handleClick = (id) => {
+    const book = { ...this.state.data };
+    book.favorited = !book.favorited;
+    this.setState({ data: book });
+    this.props.handleClick(id);
+  };
+
   render() {
+    let button = "";
+    if (
+      this.props.favorites &&
```

Comment:
you could create a variable to store this condition and give it a descriptive name, between the let button and the if statement, then use it in the if